### DISCUSSION TITLE
Request: Bungee sync

### DISCUSSION THREAD

**Discussion by 0xC1A551C** [Zrips/CMI#88] [Feature Requests]
Is it possible to have the warps/spawns sync across some bungee servers? I don't see why mysql works, but cmi setup on a second bungee server doesn't see the cmi warps, so if I do /warp on server 2 it doesn't see the warps set on server1.

**Comment by mrfloris** [comment]
Adding automatic syncing and data sharing of CMI features on a bungee network would be really helpful where CMI is the core plugin. Homes, Economy, Kits, Warps, feel logical to be the same on server A as on server B. Right now the solution for players is to turn off these features within CMI and use alternatives, which can lead to potential issues.

Personally I see this question on a weekly, if not almost daily basis on the Discord server. It's one of the top 5 highly requested features. And while I personally don't use Bungee, having the ability to centralize or properly share data between CMI servers on a network seems like a logical base feature for a plugin we use as the core of our server management.
